project2/card.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="card.css">
+    <link href="https://fonts.googleapis.com/css?family=Dosis:400,700|Source+Sans+Pro:400,900" rel="stylesheet">
+    <title>Article Card Specimen</title>
+</head>
+
+<body>
+
+    <!-- Article Image and article section-->
+    <figure class="article-image">
+        <img src="images/miley-liam.jpg" alt="Image, Miley Cyrus and Liam Hemsworth are side by side looking at each other lovingly
```

Comment:
A screenreader already announces that this is an image, before it reads the `alt` text, so you can just start with "Miley...":
```suggestion
        <img src="images/miley-liam.jpg" alt="Miley Cyrus and Liam Hemsworth are side by side looking at each other lovingly
```